### DISCUSSION TITLE
VTT and SRT subtitles with very long lines

### DISCUSSION THREAD

**Discussion by ubanning** [openai/whisper#950] [Q&A]
Hi everyone, is there any way to decrease line length in srt and vtt subtitles?
Depending on the audio, some lines can exceed 500 characters. :( 
Thanks!

**Comment by cafew** [comment]
def add_line(s):
    new_s = s
    s_count = len(s)
    s_max_count = 42
    if s_count >= s_max_count:
        if (s_count - s_max_count) >= 3:# change line if bigger than 42
            new_s = s[:s_max_count] + "\n" + s[s_max_count:]
    return new_s
